link/kprobe_multi.go

```diff
@@ -37,6 +37,9 @@ type KprobeMultiOptions struct {
 	// Each Cookie is assigned to the Symbol or Address specified at the
 	// corresponding slice index.
 	Cookies []uint64
+
+	// Probe is attach as session with BPF_TRACE_KPROBE_SESSION attach type
```

Comment:
Please try to be more helpful in API comments and pay attention to grammar. I had to search around for your upstream patch set to understand what this does. Something along the lines of:

```
// Session must be true when attaching Programs with the [ebpf.AttachTraceKprobeSession]
// attach type.
//
// This makes a Kprobe execute on both function entry and return. The entry program can
// share a cookie value with the return program and can decide whether the return program
// gets executed.
```

Reply:
sorry about that, will add more info in the comments, thanks